### ISSUE TITLE
Build issue error MSB3202: The project file "/root/webone/webone.csproj" was not found.

### ISSUE THREAD

**Issue by T0MuX** [atauenis/webone#113]
Hi, I actually already solved this one. But I have to inform you that the problem just comes from the name of the file. The build.sh script is looking for ./webone.csproj while the filename is WebOne.csproj. So, on Linux which is case sensitive, build.sh fails.
So, I just renamed WebOne.csproj into webone.csproj and now it builds.
:smile:

**Comment by atauenis** [comment]
It's a very interesting thing, as on my development machine the file is called lowercase last years. However, Git does not tracks the change. It preserving old name case, used on start of development.
![2024-02-02_22-38-26](https://github.com/atauenis/webone/assets/4800266/00d610c5-db12-4c53-9fbd-1aaba8acc3d5)

**Comment by T0MuX** [comment]
Maybe this is linked to the OS you use ? Windows right ? Windows is not case sensitive. If you use git on Windows, maybe it just ignores the case change ? I'm using Linux

**Comment by atauenis** [comment]
Yes, my primary development environment is Windows and Git, included in Visual Studio. So still looking how to rename file case using this all.

**Comment by atauenis** [comment]
Fixed via #168 .
